compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -561,16 +576,19 @@ def IREELinalgExt_AttentionOp : IREELinalgExt_Op<"attention",
     Value getValue() {
       return getDpsInputOperand(2)->get();
     }
+    Value getScale() {
+      return getDpsInputOperand(3)->get();
+    }
     Value getOutput() {
       return getDpsInitOperand(0)->get();
     }
     std::optional<Value> getMax() {
-      if (getNumResults() == 1)
+      if (getNumResults() < 2)
```

Comment:
dont understand this change. why is it not == 1? Here and below.

Reply:
I did the change to make it a bit safer to access the extra arguments--given that we can have two outputs and then the `getSum` would crash. I later found actually we have validator checks for the case. So a minor issue maybe. Feel free to revert.

Reply:
I see, that makes sense. No let's keep this as is. Thanks!